code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -46,6 +46,8 @@
 /obj/item/handcuffs/proc/can_place(var/mob/target, var/mob/user)
 	if(user == target)
 		return 1
+	if(target.lying) //Mobs that are lying down can be handcuffed without needing to grab them, to make arrests a little easier
```

Comment:
this should be target.resting, not target.lying

lying is strictly for transform and can be 0, 90, -90, etc